docs/cloudbees-ci/cbci-10.md

```diff
@@ -45,9 +45,9 @@ class: center
 ---
 name: hibernating-masters-cost-saving
 
-# How Hibernating Managed Controllers Reduce Infrastructure Costs
+# How Hibernating Managed Controllers Optimize Resources
 
-There are several ways that costs may be reduced with ***managed controller*** hibernation:
+There are several ways to optimize continuous integration resources with ***managed controller*** hibernation:
 
 1. When using Kubernetes auto-scaling and a ***managed controller*** is hibernated then the Kubernetes cluster has the potential to scale down by removing a node.
```

Comment:
are you sure you want to highlight managed controller every. single. time?

Reply:
I have been removing most of them when making other changes to a give page of content.